osbs/build/spec.py

```diff
@@ -354,8 +357,30 @@ def set_params(self, git_uri=None, git_ref=None,
         self.yum_proxy.value = yum_proxy
         self.koji_parent_build.value = koji_parent_build
 
+        if (signing_intent or compose_ids) and not self.odcs_enabled():
+            raise OsbsValidationException(
+                'signing_intent and compose_ids are allowed only when ODCS is enabled')
+
+        if signing_intent and compose_ids:
+            raise OsbsValidationException(
+                'Please only define signing_intent -OR- compose_ids, not both')
+
+        if compose_ids and yum_repourls:
+            raise OsbsValidationException(
+                'Please only define yum_repourls -OR- compose_ids, not both')
+
+        if not (compose_ids is None or isinstance(compose_ids, list)):
+            raise OsbsValidationException("compose_ids must be a list")
+        self.signing_intent.value = signing_intent
+        self.compose_ids.value = compose_ids or []
         self._populate_image_tag()
 
+    def odcs_enabled(self):
+        if self.odcs_url.value and self.odcs_openidc_secret.value:
+            return True
+        else:
+            return False
```

Comment:
Just `return self.odcs_url.value and self.odcs_openidc_secret.value`?

Reply:
ok